src/Razor/src/Microsoft.VisualStudio.RazorExtension/cgmanifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
```

Comment:
The whitespace changes here are annoying, but PowerShell doesn't let you control whitespace, and at least once this change is made, future runs of the tool will have minimal changes.